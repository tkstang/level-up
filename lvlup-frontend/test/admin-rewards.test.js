import AdminRewards from '../src/components/Admin/rewards/admin-rewards/rewards-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('AdminRewards should render', () => {
  const adminrewards = shallow(
    <AdminRewards />,
  );
  expect(toJson(adminrewards)).toMatchSnapshot();
});
