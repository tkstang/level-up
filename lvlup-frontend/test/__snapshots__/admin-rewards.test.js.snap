exports[`test AdminRewards should render 1`] = `
<lifecycle(RewardsTable)
  campusRewards={[Function]}
  makeRewardInactive={[Function]}
  resetAddReward={[Function]}
  resetEditReward={[Function]}
  resetRewardsList={[Function]}
  selectReward={[Function]}
  sortRewardCategory={[Function]}
  sortRewardCategoryReverse={[Function]}
  sortRewardName={[Function]}
  sortRewardNameReverse={[Function]}
  sortRewardPoints={[Function]}
  sortRewardPointsReverse={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "notify": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": null,
    }
  } />
`;
